src/plugins/discover/public/components/discover_grid/get_render_cell_value.tsx

```diff
@@ -214,10 +214,11 @@ function renderPopoverContent({
   }
 
   return (
-    <EuiFlexGroup gutterSize="none" direction="row">
+    <EuiFlexGroup gutterSize="none" direction="row" responsive={false}>
```

Comment:
FYI @jughosta disabled responsiveness for this, since showing the `x` below the content looks ugly :)